src/index.js

```diff
@@ -1,241 +1,267 @@
-//#############################################################################
-//#
-//# "SQIP" (pronounced \skwɪb\ like the non-magical folk of magical descent)
-//# is a SVG-based LQIP technique - https://github.com/technopagan/sqip
-//#
-//# Installation:
-//# npm install -g sqip
-//#
-//# Requirements:
-//# * Node.js >= v.6 (https://nodejs.org/en/)
-//# * Golang (https://golang.org/doc/install)
-//# * Primitive (https://github.com/fogleman/primitive)
-//#
-//#############################################################################
-
-
-//#############################################################################
-//# CONFIG
-//#############################################################################
+// #############################################################################
+// #
+// # "SQIP" (pronounced \skwɪb\ like the non-magical folk of magical descent)
+// # is a SVG-based LQIP technique - https://github.com/technopagan/sqip
+// #
+// # Installation:
+// # npm install -g sqip
+// #
+// # Requirements:
+// # * Node.js >= v.6 (https://nodejs.org/en/)
+// # * Golang (https://golang.org/doc/install)
+// # * Primitive (https://github.com/fogleman/primitive)
+// #
+// #############################################################################
+
+// #############################################################################
+// # CONFIG
+// #############################################################################
 
 // Require the necessary modules to make sqip work
-const sizeOf = require('image-size');
-const argv = require('argv');
-const os = require('os');
-const child_process = require('child_process');
-const fs = require('fs');
-const path = require('path');
-const svgo = require('svgo');
+const sizeOf = require('image-size')
+const argv = require('argv')
+const os = require('os')
+const childProcess = require('childProcess')
```

Comment:
This is going to brake, package name is `child_process`

Reply:
😱 omg yes, sorry for this one 😅